Makefile

```diff
@@ -183,7 +183,13 @@ redeploy:
 undeploy:
 	@echo
 	@echo \* Deleting any managed virtual clusters...
-	-kubectl delete ${cluster_resource_name} --all --now
+	@set -e; \
+		all_namespaces=`kubectl get ns --no-headers| awk '{print $$1}'`; \
+		echo $$all_namespaces; \
+		for ns in $$all_namespaces; do \
+			echo kubectl -n $$ns delete ${cluster_resource_name} --all --now; \
+			kubectl -n $$ns delete ${cluster_resource_name} --all --now; \
```

Comment:
Previously we had been using the "minus sign" in front of the command so that an error with the delete (I think this can even happen if there are just no clusters?) wouldn't terminate the recipe.

Since we're running a "set -e" shell here, if we want to keep the same behavior on this line we should probably do
```bash
kubectl -n $$ns delete ${cluster_resource_name} --all --now || true; \
```